OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -136,51 +412,171 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                         
                         </decoDesc>-->
                         
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="Unclear"></desc>
+                                    <locus target="#3r"/>
+                                    <q xml:lang="gez">
+                                        ዕፁበ፡ ግብረ፡ ወመድምመ፡ ኢርኢኩ፡ ወኢሰማዕኩ፡ ከዳዊት፡ ከሰሎሞን፡ ከዛቲ፡ ዘመን፡ ንጉሥ፡ ለስደት፡ ንጉሥ፡ መሸኃዳር፡ ኢርኢኩ፡ ወኢሰማዕኩ፡ 
+                                        ገዳማተ፡ ትግሬ፡ ወኢትዮጵያ፡ ብክያ፡ ወላህዋ፡ እምሥራቅ፡ እስከ፡ ምዕራብ፡ መንግሥት፡ እናጥፋ፡ አሉ፡ መሳ<add place="above">ፍ</add>ንት፡ 
+                                        ወመኳንንት፡ ዕፁበ፡ ግብረ፡ ወምድ<cb n="b"/>ምመ፡ ኢርኢኩ፡ ወኢሰማዕኩ፡ ንጉሥ፡ ለስደት፡ ንጉሥ፡ መሸኃዳር፡ ኢርኢኩ፡ ወኢሰማዕኩ፡ 
+                                        ገዳማተ፡ ትግሬ፡ ወኢትዮጵያ፡ ብክያ፡ ወላህዋ፡ 
+                                    </q>
+                                    <note>The note is written in Gǝʿǝz with some words and expressions in Amharic. The end of the note is barely legible due to a water stain.</note>
+                                </item>
+                                <item xml:id="a2">
+                                    <desc type="Unclear"></desc>
+                                    <locus target="#78va"/>
+                                    <q xml:lang="am">
+                                        <hi rend="rubric">ሐሰበ<supplied reason="omitted">፡</supplied> ፍከሬሁ<supplied reason="omitted">፡</supplied> 
+                                            ለዳዊት<supplied reason="omitted">፡</supplied> ስምአ<supplied reason="omitted">፡</supplied> ሞተ<supplied reason="omitted">፡</supplied></hi>
+                                        ምሕረት<supplied reason="omitted">፡</supplied> ወንጌላዊ<supplied reason="omitted">፡</supplied> ወርህን<supplied reason="omitted">፡</supplied> 
+                                        በምቶታ<supplied reason="omitted">፡</supplied> ፶ግደፍ፡ ደግሞሞ<supplied reason="omitted">፡</supplied> ሐባርን<supplied reason="omitted">፡</supplied> 
+                                        ሰምን፻፶ግደፍ፡ የወጸውን፡ የዳዊቱን
+                                    </q>
+                                    <note>The note is crudely written in Gǝʿǝz with some words and expressions in Amharic.</note>
+                                </item>
+                                <item xml:id="a3">
+                                    <desc type="GuestText">Miracle of Jesus Christ</desc>
+                                    <locus from="78vb" to="80rb"/>
+                                    <q xml:lang="gez">
+                                        <seg part="I">
+                                        <hi rend="rubric">ተአምረ፡ ዘገ<supplied reason="omitted">ብ</supplied>ረ፡</hi> እግዚእነ፡ ወአምላክነ፡ ኢየሱስ፡ ክርስቶ<supplied reason="omitted">ስ</supplied>
```

Comment:
I think in the case of miracles it would be useful to assign an ID, if one doesn't already exist (if there is enough text to allow you so)